kimetaro.py

```diff
@@ -9,6 +10,8 @@
 
 # Value initialization
 COMMAND_SUFFIX = ''
+PATTERN = r"(?<=\"|')(?!,).*?(?=\"|')"
```

Comment:
It's better to add a comment that shows the match pattern

Reply:
Added the comment that explains what keywords match.